src/components/Navigation.tsx

```diff
@@ -12,6 +15,28 @@ interface NavProps { user: User | null, session: Session | null }
 
 export default function Navigation(props: NavProps): JSX.Element {
 
+	const [themeIcon, setThemeIcon] = useState(<DarkModeIcon />);
+	useEffect(() => {
+		if (localStorage.theme === 'dark' || (!('theme' in localStorage) && window.matchMedia('(prefers-color-scheme: dark)').matches)) {
```

Comment:
How are we able to access the value with dot notation rather than `localStorage.getItem` and `setItem`? I see this is how the tailwind docs show but I have never seen it used like this. https://tailwindcss.com/docs/dark-mode#supporting-system-preference-and-manual-selection

Reply:
To my knowledge dot notation for `localStorage` acts as a shorthand for `getItem` and `setItem` where you can assign/read the key:value pair. We are just not able to use dot notation for `removeItem`.